exasol/analytics/audit/audit.py

```diff
@@ -0,0 +1,51 @@
+from inspect import cleandoc
+from typing import List
+
+from exasol.analytics.audit.columns import BaseAuditColumns
+from exasol.analytics.schema import (
+    Column,
+    SchemaName,
+    TableLikeName,
+    TableLikeNameImpl,
+)
+
+
+class TableDescription:
```

Comment:
Should this class go into the `schema`? It has nothing specific to the audit.

Reply:
I can follow your thoughts and tend to agree. 
Although, package schema 
* is all about names and types and 
* contains nearly no code for rendering the objects, yet.
* does not deal with values or statements, for example

Reply:
Moved to `schema.table.Table`